### PR TITLE
Get rid of dangling ES5 code

### LINKED ISSUE #36: Migrate webpack to typescript
Just so this
image
makes it to 100% 💆‍♀️

### LINKED ISSUE #50: Migrate styling to scss
Just why not at this point

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #36 
resolves #50 

No more plain js!
No more plain css!

### CODE REVIEW

**Comment by braxtonhall** [comment]
WAIT I forgot to remove the sass directory from the build

File: extension/sass/toast.sass [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	padding: 16px
+	position: fixed
+	z-index: 99
+	left: 70%
+	top: 80px
+	color: #000
+	visibility: visible
+
+	&.ready
+		pointer-events: auto
```
Comment (adirar111) [inline extension/sass/toast.sass:19]: sassy 💅

File: webpack.config.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
-const path = require('path');
+import path from "path";
+import {Configuration} from "webpack";
```
Comment (adirar111) [inline webpack.config.ts:2]: :wave: js

File: extension/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (adirar111) [inline extension/tsconfig.json:1]: wait why are there 2 of these

File: tsconfig.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
 {
 	"compilerOptions": {
+		"esModuleInterop": true,
 		"strict": false,
 		"types": [
-			"chrome"
+			"node"
```
Comment (adirar111) [inline tsconfig.json:6]: why "node" here and "chrome" in the inner one? also where are we using node types

**Review by adirar111 — APPROVED** [review]
🧼 🚰

File: tsconfig.json (hunk 4 above)
Comment (braxtonhall) [reply tsconfig.json:6]: we use node to execute webpack!

Comment (adirar111) [reply tsconfig.json:6]: oop okey

**Pull request merged**